marklogic-data-hub-central/ui/src/components/sidebar/sidebar-rtl.test.tsx

```diff
@@ -35,12 +34,44 @@ describe("Sidebar createdOn face time window dropdown", () => {
             facetRender = {jest.fn()}
             checkFacetRender = {jest.fn()}
         />);
-        userEvent.click(getByText('Hub Properties'));
         expect(getByText("Select time")).toBeInTheDocument();
         userEvent.click(getByText('select time'));
         expect(getByText("Custom")).toBeInTheDocument();
         fireEvent.click(getByText("Custom"));
         expect(getByPlaceholderText("Start date")).toBeInTheDocument();
         expect(getByPlaceholderText("End date")).toBeInTheDocument();
     });
-});
\ No newline at end of file
+
+    test('Verify that hub properties is expanded by default', () => {
+        const parsedModelData = entityFromJSON(modelResponse);
+        const entityDefArray = entityParser(parsedModelData);
+        const { getByText,querySelector } = render(<Sidebar
+            entityDefArray={entityDefArray}
+            facets={searchPayloadFacets}
+            selectedEntities={[]}
+            facetRender = {jest.fn()}
+            checkFacetRender = {jest.fn()}
+        />);
+        expect(document.querySelector('#hub-properties div')).toHaveAttribute('aria-expanded','true');
+        userEvent.click(getByText('Hub Properties'));
+        expect(document.querySelector('#hub-properties div')).toHaveAttribute('aria-expanded','false');
+    });
+
+    test('Verify that entity properties is expanded when entity is selected', () => {
+        const parsedModelData = entityFromJSON(modelResponse);
+        const entityDefArray = entityParser(parsedModelData);
+        const { getByText,querySelector } = render(<Sidebar
+            entityDefArray={entityDefArray}
+            facets={searchPayloadFacets}
+            selectedEntities={['Customer']}
+            facetRender = {jest.fn()}
+            checkFacetRender = {jest.fn()}
+        />);
+        expect(document.querySelector('#entity-properties div')).toHaveAttribute('aria-expanded','true');
+        expect(document.querySelector('#hub-properties div')).toHaveAttribute('aria-expanded','false');
```

Comment:
is it expected for hub-properties to be collapsed when an entity is selected?

Reply:
Yes, that is expected. We just need to change hub properties to be expand by default as part of this story while keeping rest of functionality same. This is how it is working on nightly instance too.

Reply:
*hub properties to be expand by default when all entities is selected